### ISSUE TITLE
Failure Following DISABLE

### ISSUE THREAD

**Issue by scruffidog** [alex-courtis/way-displays#101]
So I'm using way-displays to control screen idle action via swayidle:

swayidle -w timeout 150 "way-displays -s DISABLED eDP-1" resume "way-displays -d DISABLED eDP-1"

this worked fine until the upgrade a few days ago but now, it tells me display is disabled, even though it clearly is not:
```
Client sending request: set
  Disabled:
    eDP-1

Server received request: set
  Disabled:
    eDP-1

New configuration:
  Arrange in a ROW aligned at the TOP
  Scaling: ON
  Auto scale: OFF
  Disabled:
    eDP-1
ERROR: way-displays not running, check $XDG_VTNR
```

the last line happens when  the resume is supposed to kick in. Other than this hiccup, this has been a pretty awesome and handy utility!

**Comment by alex-courtis** [comment]
It looks like the server died when setting DISABLED. I'll look into that.

Which version are you running? `way-displays --version`

Has this ever worked or is this a new problem?

**Comment by alex-courtis** [comment]
regression at 393fdd8

**Comment by alex-courtis** [comment]
This should be resolved by 1.8.1.

Please watch [Package Manager](https://github.com/alex-courtis/way-displays#package-manager) for your distribution.

**Comment by scruffidog** [comment]
sorry, was out and about. This was working and stable on version 1.7.1. It started with 1.8.0 and I just installed 1.8.1 which seems to resolve the issue. Thanks for being so on top of things!
